### ISSUE TITLE
Host on gh-pages

### ISSUE THREAD

**Issue by KeizerDev** [ManrajGrover/SingleDivProject#45]
Move branch to gh-pages branch and host it all there using iframes.

**Comment by ManrajGrover** [comment]
@KeizerDev Yes, we definitely need pages to host this. As per #44 , we need to first unify flag sizes before creating such page. Thank you for suggesting this.

**Comment by module17** [comment]
@ManrajGrover @KeizerDev If we host on gh-pages we could get SVG support working properly perhaps...
